### ISSUE TITLE
03_pytorch_computer_vision  clarification on a comment

### ISSUE THREAD

**Issue by samiulhq** [mrdbourke/pytorch-deep-learning#886]
# 2. Calculate loss (per batch)
        loss = loss_fn(y_pred, y)
        train_loss += loss # accumulatively add up the loss per epoch 

I believe this is accumulatively adding up the loss per batch to get the total loss per epoch.
